src/motion_planning/scripts/task.py

```diff
@@ -2,11 +2,11 @@
 from nav_msgs.msg import Odometry
 from geometry_msgs.msg import Pose, Twist
 
-from abc import ABC, abstractmethod
+from abc import ABCMeta, abstractmethod
 
-class Task(ABC):
+class Task:
     """High level task that represents some function"""
-
+    __metaclass__ = ABCMeta
```

Comment:
Why do you choose to do it this way instead of inheritance from ABC?

Reply:
You can't inherit from ABC in python 2

Reply:
Got it